### PR TITLE
Check documentation

### PR DESCRIPTION
**Pull request by nojaf**
More of the same 😽

### CODE REVIEW

File: docs/docs/end-users/FormattingCheck.md [hunk 1]
```diff
@@ -6,21 +6,116 @@ index: 5
 
 # Formatting Check
 
+Formatting source code is a habit, a step in your developer workflow.  
+The benefits of consistently formatting is that **your delta** (*typically the changes files in a pull request*) will be **smallest set possible** if every previous change was formatted.
```
Comment (dawedawe) [inline docs/docs/end-users/FormattingCheck.md:10]:
```suggestion
The benefits of consistently formatting is that **your delta** (*typically the changes in the files of a pull request*) will be the **smallest set possible** if every previous change was formatted.
```

File: docs/docs/end-users/FormattingCheck.md [hunk 2]
```diff
@@ -6,21 +6,116 @@ index: 5
...
+If you agree as a team that you need to write unit tests to guarantee the code quality, you expect every developer to cover any new code with a test.  
+Formatting is no different, every developer should do it and it is an acceptance criteria for new code.
+
+Imagine we have multiple developers in our team. Hektor made the initial setup and during a team meeting it was decided that the code should always be formatted.  
+Once the initial project structure was delivered, Achilles made a change where the code was not formatted.  
+The next day, Odysseus wants to submit a new pull request. As agreed code should be formatted, so Odysseus did exactly that.
+As the changes Achilles made were not formatted, there was more code touched than was absolutely necessary. Perseus didn't see any harm in this rectification and merge the pull request as is.
+
+A week later, a huge problem was discovered in production. There was data loss and the entire company was on fire.
+Fuelled by rage and anger Hektor wanted to know who was responsible for this devastating tragedy. You guessed it, somebodies head was about to roll.
```
Comment (dawedawe) [inline docs/docs/end-users/FormattingCheck.md:24]:
```suggestion
Fuelled by rage and anger Hektor wanted to know who was responsible for this devastating tragedy. You guessed it, somebody's head was about to roll.
```

File: docs/docs/end-users/FormattingCheck.md [hunk 3]
```diff
@@ -6,21 +6,116 @@ index: 5
...
+Perseus was tasked with getting to the bottom of this mystery and eventually he located the source of the misery.  Hektor tasked him with running a `git blame` command, to see who the culprit really was.
+
+Odysseus credentials showed up, but it was the function Achilles originally wrote. Unable to escape Hektor's wrath, Odysseus was fired immediately.
+Achilles felt bad but couldn't come clean as he was about to get surgery for his heel. Odysseus took the fall like a true hero.
+But it lead him to downwards spiral and would take him ten years before he could land another job in the software industry.
```
Comment (dawedawe) [inline docs/docs/end-users/FormattingCheck.md:29]:
```suggestion
But it lead him to a downwards spiral and would take him ten years before he could land another job in the software industry.
```

File: docs/docs/end-users/FormattingCheck.md [hunk 4]
```diff
@@ -6,21 +6,116 @@ index: 5
...
+
+The recommendation is to install `fantomas` as a local tool and run it using the generic `DotNet.exec` api.
+This translates to running `dotnet fantomas ourSourceFolder --recurse --check`.
+
+### Any other continuous integration environment
+
+You want to restore your local `fantomas` tool using `dotnet tool restore`.
+Next, you want to run `dotnet fantomas <input> --check` and make sure your continuous integration environment **fails** your job when **a non-zero exit code** is returned.
+
+**Pro-tip**: print the command users need to run to fix the formatting in the output log when the check failed. This is useful for open-source projects where new contributors might never have exposed to formatting.
```
Comment (dawedawe) [inline docs/docs/end-users/FormattingCheck.md:93]:
```suggestion
**Pro-tip**: print the command users need to run to fix the formatting in the output log when the check failed. This is useful for open-source projects where new contributors might never have been exposed to formatting.
```

**Review by dawedawe — APPROVED** [review]

**Comment by dawedawe** [comment]
If the Greek tragedy doesn't convince people to use Fantomas, than I don't know what can.

**Comment by nojaf** [comment]
Thanks for the review @dawedawe !

**Pull request merged**